test/completion/mn.exp

```diff
@@ -0,0 +1 @@
+assert_source_completions mn
```

Comment:
Please add an actual test case. doc/testing.txt contains some instructions for generating a bare bones one.